docs/content/usage/scenarios/custom-scenario/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: " Custom"
+date: 2021-07-28T14:36:02+05:30
+description: "Configure a custom scenario to test your network's defenses."
+weight: 100
+pre: "<i class='fas fa-edit'></i>"
+chapter: true
+---
+
+# Custom
+
+If none of the existing scenarios fit your use case, you can choose the "Custom" scenario and configure the simulation according to your needs.
```

Comment:
I'd say that choosing custom is not only for cases when "none of the existing scenarios fit your use case", but also for cases when you want additional functionality. I would reword to something like:
"The Infection Monkey is a versatile breach and attack simulation tool. Choose a "Custom" scenario to access all of its capabilities. This will allow you to enhance, optimize and fine-tune the simulation."